Cargo.toml

```diff
@@ -122,6 +122,7 @@ members = [
     "misc/metrics",
     "misc/multistream-select",
     "misc/peer-id-generator",
```

Comment:
Would there still be any use for `peer-id-generator`? In other words, how about removing it within this pull request?